packages/react/src/helpers/arguments.ts

```diff
@@ -78,3 +81,69 @@ export function useMediaTransformFromConfig<TVariables extends OperationVariable
     ...mediaTransformConfigToQueryVariables(mediaTransforms),
   };
 }
+
+export type WithObserverIdOverride<TVariables = unknown> = Prettify<
```

Comment:
most of these helpers are not relevant in lens v2 as there is no concept of observer, is any of them in use?